h2o-core/src/main/java/water/FailedNodeWatchdogExtension.java

```diff
@@ -24,7 +26,15 @@ public void printHelp() {
                         "    -watchdog_client\n" +
                         "          Same as the client except the that cluster is stopped when this client \n" +
                         "          disconnects from the rest of the cloud or the cloud is stopped when it doesn't \n" +
-                        "          hear heartbeat from the client for specified amount of time."
+                        "          hear heartbeat from the client for specified amount of time. \n" +
+                        "    -watchdog_client_connect_timeout\n" +
```

Comment:
What are use-cases - when i will need to use the first one, when the second one?